src/main/resources/migrations/00043_create_table_oauth_states.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-gocardless_partner_app_account_connect_tokens
+CREATE TABLE gocardless_partner_app_account_connect_tokens (
+  id BIGSERIAL PRIMARY KEY,
+  gateway_account_id BIGSERIAL NOT NULL,
```

Comment:
I think we just want `BIGINT` here: we’ll be specifying the value every time rather than relying on a sequence providing us with one